### PR TITLE
Fixes for tests

### PR DESCRIPTION
**Pull request by alneberg**

### CODE REVIEW

File: taca/element/Element_Runs.py [hunk 1]
```diff
@@ -149,19 +158,19 @@ def check_sequencing_status(self):
...
+            ):
                 return "ongoing"
             elif os.path.exists(self.demux_dir) and os.path.isfile(
                 os.path.join(demux_dir, self.demux_stats_file)
-                ):
-                finished_count += 1  # TODO: check exit status of demux in exit status file
+            ):
+                finished_count += (
+                    1  # TODO: check exit status of demux in exit status file
+                )
```
Comment (ssjunnebo) [inline taca/element/Element_Runs.py:173]: Just out of curiosity, why the parenthesis?

File: tests/element/test_Element_Runs.py [hunk 2]
```diff
@@ -101,10 +106,14 @@ def test_check_sequencing_status(
...
     def test_get_demultiplexing_status(
-        self, p: pytest.fixture, create_dirs: pytest.fixture
+        self, mock_db: mock.Mock, p: pytest.fixture, create_dirs: pytest.fixture
     ):
         tmp: tempfile.TemporaryDirectory = create_dirs
 
+        if p["demux_dir"] and not p["demux_done"]:
+            import pdb
+
+            pdb.set_trace()
```
Comment (ssjunnebo) [inline tests/element/test_Element_Runs.py:116]: Do you want to keep this?

**Review by ssjunnebo — APPROVED** [review]
Looks good, thanks!

File: taca/element/Element_Runs.py (hunk 1 above)
Comment (alneberg) [reply taca/element/Element_Runs.py:173]: This one was Ruff I think. I think it does it when it needs newlines to not go over the limit

File: tests/element/test_Element_Runs.py (hunk 2 above)
Comment (alneberg) [reply tests/element/test_Element_Runs.py:116]: No, please remove 😂

Comment (ssjunnebo) [reply tests/element/test_Element_Runs.py:116]: Done 😄

**Pull request merged**